docs/faq/README.md

```diff
@@ -86,3 +87,96 @@ To get the latest version, drop this into the HTML:
 ```
 
 We also recommend changing the JavaScript language to ES6/Babel.
+
+## Can I use my favourite CSS/LESS/SASS syntax?
+
+Yes, you can, with the help of the [react-styling](https://github.com/halt-hammerzeit/react-styling) module. It requires you to write your code in ES6/Babel (everyone does it nowadays). Using [react-styling](https://github.com/halt-hammerzeit/react-styling) module you can write your styles in any syntax you like (curly braces or tabs, CSS, LESS/SASS, anything will do). Below are some examples of how can standard Radium examples be rewritten using this module.
+
+The example from the main Readme
+
+```as
+<Button kind="primary">Radium Button</Button>
+```
+
+```as
+@Radium
+class Button extends React.Component {
+  static propTypes = {
+    kind: React.PropTypes.oneOf(['primary', 'warning']).isRequired
+  }
+
+  render() {
+    return (
+      <button style={style.button[this.props.kind]}>
+        {this.props.children}
+      </button>
+    )
+  }
+}
+
+const style = styler
+`
+  button
```

Comment:
Use regular CSS syntax with the first example (with braces and semicolons) so people aren't too confused